src/Test.Utilities/CSharpCodeFixVerifier`2.cs

```diff
@@ -33,13 +34,13 @@ public static async Task VerifyAnalyzerAsync(string source, params DiagnosticRes
             await test.RunAsync();
         }
 
-        public static async Task VerifyCodeFixAsync(string source, string fixedSource)
+        public static async Task VerifyCodeFixAsync([StringSyntax("C#-test")] string source, string fixedSource)
```

Comment:
Might be worth adding the attribute also on the `string fixedSource`.

Reply:
Thanks! Not sure how I missed these in the original changes, `src/Test.Utilities/CSharpSecurityCodeFixVerifier`2.cs` had them on the `fixedSource`  parameters already.